### PR TITLE
Add Inspector Java SDK reference

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

File: pages/implementation/inspector/sdk/java.mdx [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+> - `String appName` - application name
+> - `AvoInspectorEnv env` - current environment: AvoInspectorEnv.Dev, AvoInspectorEnv.Staging or AvoInspectorEnv.Prod
+
+## <a name="event-tracking"></a>Sending event schemas to Avo Inspector
+
+This is the core of the **Avo Inspector SDK**.
+
+Call \***\*one of the methods\*\*** in this section every time an event is tracked.
+
+### Option 1
```
Comment (bjornj12) [inline pages/implementation/inspector/sdk/java.mdx:66]: What option do we recommend? Can we set up a "analytics wrapper" example with our recommended approach? 

```public void trackEvent(name, properties)
{
   trackingService.trackEvent(eventName, properties);
   Inspector.trackschema(eventName, properties);
}

Dno, something in that direction?

Comment (alex-tpom6oh) [reply pages/implementation/inspector/sdk/java.mdx:66]: First option covers most of the use cases, like the wrapper function

Comment (bjornj12) [reply pages/implementation/inspector/sdk/java.mdx:66]: yep, Im thinking of an example like this for them to relate to, so they don't have to think 
"How do i set this up into my code?"

Using option1, if we can create a example wrapper function so for 99% of java users coming in, they can just copy paste without reading too much on the docs and it just works

Comment (alex-tpom6oh) [reply pages/implementation/inspector/sdk/java.mdx:66]: Right, I was not including the examples in the reference docs on purpose, but it probably makes sense to include

Comment (alex-tpom6oh) [reply pages/implementation/inspector/sdk/java.mdx:66]: Added a wrapper example

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…nspector-sdk-reference**](https://docs-dflvfcxzd.avo-dev.app/implementation/avo-inspector-sdk-reference) | [**…ion/inspector/sdk/java**](https://docs-dflvfcxzd.avo-dev.app/implementation/inspector/sdk/java) |
|:-:|:-:|
|<a href="https://docs-dflvfcxzd.avo-dev.app/implementation/avo-inspector-sdk-reference">Screenshot of /implementation/avo-inspector-sdk-reference</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/c-pcrJoygKcPFdTjr8u7itgUB4FU8dsNJE3t3w-m7Ttr2HUOquHdH1E4_52U7wKhripROuS5yoA-qsyuJwXlIg.png">(view full size)</a> |<a href="https://docs-dflvfcxzd.avo-dev.app/implementation/inspector/sdk/java">Screenshot of /implementation/inspector/sdk/java</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/DifV7Mz9nFCbm5rIPjTB4ziZ7ThMmKb-hITA8zPBHMadcByWbvQdn8ssOeM85nXPhQMETcMElCet17MHgiBWgQ.png">(view full size)</a> |



| [**…ntation/reference/java**](https://docs-dflvfcxzd.avo-dev.app/implementation/reference/java) | [**…on/setup-inspector-sdk**](https://docs-dflvfcxzd.avo-dev.app/implementation/setup-inspector-sdk) |
|:-:|:-:|
|<a href="https://docs-dflvfcxzd.avo-dev.app/implementation/reference/java">Screenshot of /implementation/reference/java</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/G4EyMC3-Jm2mbrV-vfLVAzJmJgIMraHbdoSKLWfXrwOx9rmROFQEqp73M5YSqOsUCxRjnZo4_zsuacSo9qSwYw.png">(view full size)</a> |<a href="https://docs-dflvfcxzd.avo-dev.app/implementation/setup-inspector-sdk">Screenshot of /implementation/setup-inspector-sdk</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/I5Yk1Bxo5cJmPDcR0kXVFXhawE--QRqGZGYJrxJtJcNAx0xEDa861dOrUTaPAghg4iYMlJo4IKXq0CcMUhrmlg.png">(view full size)</a> |







Commit 4af3fcc7810d1d862809414cb64983c7e7b6ce7e (https://docs-dflvfcxzd.avo-dev.app).

**Review by bjornj12 — APPROVED** [review]

File: pages/implementation/reference/java.mdx [hunk 2]
```diff
@@ -65,7 +65,7 @@ Avo.signupStart("direct")
 
 #### Step 4. Verify the implementation
 
-Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/android"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a>.
+Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the Avo Inspector to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/java.mdx:68]:
```suggestion
Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the Avo Inspector to verify that your implementation is correct. If you don't want to use the <Link passHref href="/implementation/inspector/sdk/java"><a>Avo Inspector</a></Link> you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```

File: pages/implementation/reference/java.mdx [hunk 3]
```diff
@@ -65,7 +65,7 @@ Avo.signupStart("direct")
 
 #### Step 4. Verify the implementation
 
-Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/android"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a>.
+Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the Avo Inspector to verify that your implementation is correct. If you don't want to use the <Link passHref href="/implementation/inspector/sdk/java"><a>Avo Inspector</a></Link> you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/java.mdx:68]:
```suggestion
Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/java"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```

File: pages/implementation/reference/java.mdx [hunk 4]
```diff
@@ -65,7 +65,7 @@ Avo.signupStart("direct")
 
 #### Step 4. Verify the implementation
 
-Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/android"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a>.
+Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/java"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```
Comment (alex-tpom6oh) [inline pages/implementation/reference/java.mdx:68]:
```suggestion
Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a> in your client applications.
```

**Pull request merged**